packages/client/lib/client/index.ts

```diff
@@ -163,6 +167,9 @@ export default class RedisClient<
   static #createModuleCommand(command: Command, resp: RespVersions) {
     const transformReply = getTransformReply(command, resp);
     return async function (this: NamespaceProxyClient, ...args: Array<unknown>) {
+      if (command.unstableResp3Module && resp == 3 && !this._self._self.#options?.unstableResp3Modules) {
```

Comment:
move the if + throw to a function and reuse it everywhere?

Reply:
as discussed on zoom, moved this to how the command is actually attached to object in attachCommand